app/addons/fauxton/components.js

```diff
@@ -139,17 +134,24 @@ function(app, FauxtonAPI, ace, spin, ZeroClipboard) {
       };
 
       return {
+        toggleDisabled: this.toggleDisabled,
         crumbs: crumbs,
         nextCrumbHasLabel: nextCrumbHasLabel
       };
     },
 
     toggleLastElement: function (event) {
+      if (this.toggleDisabled) {
+        return;
+      }
```

Comment:
click-events + mouseover events where hardwired into the breadcrumb component, had to make it configurable for using it as simple breadcrumb